modules/libfuse/3.16.2/presubmit.yml

```diff
@@ -0,0 +1,8 @@
+matrix:
+  platform: ["ubuntu2204"]
```

Comment:
nit: any reason to not support windows and macos?

Reply:
Fuse is very much a Linux thing. There is https://github.com/billziss-gh/winfuse and https://osxfuse.github.io/ but these are separate projects